lib/fs/modules/LinuxMount.rb

```diff
@@ -309,23 +309,27 @@ def expandLinks(p)
     #
     components.each do |c|
       ncp = File.join(cp, c)
-      #
-      # Each file system know how to check for,
-      # and read its own links.
-      #
-      fs, lp = getFsPathBase(ncp)
-      if fs.fileSymLink?(lp)
-        sl = getSymLink(fs, lp)
-        if sl[0, 1] == '/'
-          cp = sl
-        else
-          cp = File.join(cp, sl)
-        end
+      cp = follow_all_symlinks(ncp)
+    end
+    (cp)
```

Comment:
I know this was like this before your change, but can you remove the parenthesis?

Reply:
Yup.